### ISSUE TITLE
Resources and Support audit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17365]
## Description
Complete a full accessibility audit of the Resources and Support product, including:
- [Landing page](https://www.va.gov/resources/)
- [Taxonomy page](https://www.va.gov/resources/va-account-and-profile/)
- [Search results](https://www.va.gov/resources/search/?query=disability)
- Detail pages
  - https://www.va.gov/resources/non-compensable-disability/
  - https://www.va.gov/resources/how-are-pension-benefits-and-disability-compensation-different/
  - https://www.va.gov/resources/military-sexual-trauma-and-disability-compensation/
  - https://www.va.gov/resources/separation-health-assessment-for-service-members/

## Acceptance Criteria

- [x] Accessibility audit has been completed
- [x] Recommended updates and improvements have been documented
- [x] Any high priority/critical updates are brought to the attention of PM/DM/PO
- [x] Design review ticket is created

**Comment by laflannery** [comment]
### Accessibility Updates

- Search button on mobile:
  - This is missing accessibility attributes, we need aria-expanded/ aria-collapsed, and aria-controls
- Duplicate links for “Go to all articles”
  - Can we update these to be “Go to all {type} articles”?
- Headings
  - The search section is above the article on detail pages which means the H1 is not the first heading on the page here
  - We should think about redoing this, moving the search of removing the header from the legend
- Search section
  - The search field is not contained within the fieldset and it would make more sense if it were because it is all one group
    - Can a search component be put into a radio component?
  - Do we need a visual label here “Enter keyword, phrase or question”? The component doesn’t allow this so we are doing it in a sort of complicated way and so:
    - If we need this, let’s chat with DST to see if adding this is a possibility
    - If we don’t let’s remove
  - Component itself adds both SR-only text and an aria-label. Why? It’s just unnecessary
- Show all topics button
  - Focus should be brought to the “Life Insurance” heading

### Components

- [Breadcrumbs](https://design.va.gov/components/breadcrumbs)
  - These are currently not using a component and will be updated by the DST soon-ish
- [Need more help?](https://design.va.gov/components/form/need-help)
  - Should this section be a component?
- [Link - Collection](https://design.va.gov/components/link/collection)
  - Should the Related information section be a [Related links](https://design.va.gov/components/link/collection#related-links) component?
- [Tag](https://design.va.gov/components/tag)
  - Our usage does not match with the DS component so these are “imposters”. What should we do with these?

### Other updates/improvements

- Page width
  - Is it weird that the middle of the page has a wider width than the top and bottom?
- Accordion Component questions:
  - Should the expand all show if there is only one?
  - Or is this more a content concern where we don't need to be using an accordion if there is only one?
-
